### DISCUSSION TITLE
upgrade assistance with your fix.

### DISCUSSION THREAD

**Discussion by ap-coder** [spatie/laravel-medialibrary#2303] [Q&A] [answered]
This is probably a dump question but where do i put this code for it to loop or run.  I am upgrading and not sure how to use it. 

```
use Spatie\MediaLibrary\MediaCollections\Models\Media;

Media::cursor()->each(
   fn (Media $media) => $media->update(['uuid' => Str::uuid()])
);

```

**Comment by 4unkur** [comment, marked as answer]
You have to run this 1 time, so you can use tinker for that (`php artisan tinker`)  
Or create migration with this code inside.
